lib/resque/plugins/retry/hooks.rb

```diff
@@ -0,0 +1,62 @@
+module Resque
```

Comment:
The only reason I pulled into a separate module was just to keep the main module from getting bloated (not that it's that big right now). I'm fine either way; there was just some logic here that was reasonable to pull out.

I think it would be better to go with "hooks" and to go with the current method, simply to be consistent with the way both resque and resque-scheduler have named and implemented them. Those two projects (at least resque, didn't check for resque-scheduler) invoke the hooks in alphabetical order, which allows the user to control the ordering. It may not be ideal, but I think it's minor enough of an issue that I would personally think it's better to be consistent with the main project.